05_stringer/CameronSchafer/main.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
this file should be in a different pull request

Reply:
I somehow added lab3 files in my lab5 pull request but that is fixed now.